packages/libs/src/sdk/api/generated/full/apis/TracksApi.ts

```diff
@@ -323,78 +326,79 @@ export class TracksApi extends runtime.BaseAPI {
 
     /**
      * @hidden
-     * Gets gated track signatures for passed in gated track ids
+     * Gets the tracks found on the \"Most Loved\" smart playlist
```

Comment:
Sorry about the cursed diff. The rename moved the method down one spot in the list and git is confused.